openslides/core/migrations/0007_auto_20180130_1400.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-01-30 13:00
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Permission
+from django.db import migrations
+
+
+def delete_old_logo_permission(apps, schema_editor):
+    """
+    Deletes the old 'can_manage_logo' permission which is replaced with
+    'can_manage_logos_and_fonts'. If this is a fresh database, no permission
+    will be deleted, in fact the old permission does not exist. Django creates
+    the permission after all migration and the old one is not generated.
+    If this is an old database, the new permission will be created and the old
+    one deleted.
+    """
+    Permission.objects.filter(codename='can_manage_logos').delete()
```

Comment:
Now a migration admin can not manage logos any more.

(Sorry. But I nevertheless think it is the right way to change perms in this file. Maybe you have to do it manually: First get all groups with the old perm. Add the new perm to these groups. Then remove the old perm.)

Reply:
It's fine. First I thought this does not work, but it does. The reassignment of the permissions is a great idea!

Reply:
Hmm, I cannot assign the new permission here. Doesn't django create the permissions after all migrations, does it? So I can filter all groups with the permission, but cannot assign the new one, because it does not exist yet..

Reply:
@normanjaeckel whats your opinion about this: https://stackoverflow.com/questions/31735042/adding-django-admin-permissions-in-a-migration-permission-matching-query-does-n/31735043

Reply:
Maybe this is the way.